app/javascript/src/hyacinth_v1/App.js

```diff
@@ -1,63 +1,14 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
 import { QueryParamProvider } from 'use-query-params';
-import { ApolloProvider } from '@apollo/react-hooks';
-import axios from 'axios';
 
 import MainContent from './MainContent';
-import createApolloClient from './utils/createApolloClient';
 
 function App() {
-  const [loading, setLoading] = useState(true);
```

Comment:
Most of the below has been extracted into `app/javascript/src/hyacinth_v1/utils/getGraphqlSchemaTypes.js`, which is now called in an event handler added in the main entry file `app/javascript/packs/hyacinth_v1.js`

Reply:
👍